chirpstack/src/config.rs

```diff
@@ -89,6 +90,31 @@ impl Default for Redis {
     }
 }
 
+#[derive(Serialize, Deserialize, Clone)]
+#[serde(default)]
+pub struct Sqlite {
+    pub path: String,
+    pub max_open_connections: u32,
+    #[serde(default = "default_sqlite_busy_timeout")]
+    pub busy_timeout: u32,
+    pub journal_mode: Option<String>,
+}
+
+impl Default for Sqlite {
+    fn default() -> Self {
+        Sqlite {
+            path: "chirpstack.sqlite".into(),
+            max_open_connections: 4,
+            busy_timeout: default_sqlite_busy_timeout(),
```

Comment:
There is no need to put the default in a function. The `#[serde(default)]` will already make sure that undefined config options are filled with the value from `default()`.